source/compiler/qsc_qasm/src/compiler.rs

```diff
@@ -114,6 +114,7 @@ pub fn compile_to_qsharp_ast_with_config(
 pub enum PragmaKind {
     QdkBoxOpen,
     QdkBoxClose,
+    QdkProfile,
```

Comment:
To match others, should be QdkQirProfile

Reply:
Done